ietf/templates/nomcom/nomcom_public_base.html

```diff
@@ -81,4 +81,4 @@ <h1>
                     });
             });
     </script>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
Please change your editor to not automatically change the end of file newline presence

Reply:
(and revert this file since there are no  other changes in it?)